tests/components/overseerr/test_coordinator.py

```diff
@@ -0,0 +1,52 @@
+"""Test the OverseerrUpdateCoordinator."""
```

Comment:
we don't usually test internals like like coordinator. Rather write tests showing the side effects of those errors (e.g. sensor going unavailable)